src/models/jwtPermissions/index.js

```diff
@@ -120,7 +120,7 @@ exports.checkPermissionsMiddleWare = (permissions) => async (ctx, next) => {
  * @param {string} tokenPermissions
  * @param {number[]|number} permissions
  */
-const checkAtLeastOnePermissions = (tokenPermissions, permissions) => {
+const checkAtLeastOnePermission = (tokenPermissions, permissions) => {
     const decodedPermissions = decodePermissions(tokenPermissions);
     const validPermission = permissions.find((i) => decodedPermissions[i]);
```

Comment:
this will not work for permissions with index 0

Reply:
done